src/bundle/Resources/config/services/ui_config/common.yml

```diff
@@ -100,3 +100,7 @@ services:
     EzSystems\EzPlatformAdminUiBundle\Templating\Twig\PathStringExtension: ~
 
     EzSystems\EzPlatformAdminUiBundle\Templating\Twig\ContentTypeIconExtension: ~
+
+    EzSystems\EzPlatformAdminUiBundle\Templating\Twig\DateTime\DateTimeExtension: ~
```

Comment:
It shouldn't be placed in this file. This namespace is reserved for `EzSystems\EzPlatformAdminUi\UI\Config` stuff. Someone has already put non relevant service definitions here (`PathStringExtension` and `ContentTypeIconExtension`).